### DISCUSSION TITLE
Why do I need to specify the full path `/usr/local/bin/alacritty` instead of just `alacritty`

### DISCUSSION THREAD

**Discussion by zedkaido** [qutebrowser/qutebrowser#7841] [Q&A] [answered]
I am on a OSX Monterey and am using the latest qutebrowser brew formula.

qutebrowser: v.3.0.0
Qt: 6.5.2

I noticed that if I open the `Qutebrowser.app`, I need to specify the full path `/usr/local/bin/alacritty` in order for the terminal to open. However if I open `qutebrowser` from within the terminal everything works just fine with just `alacritty`.

The following config results in the following error message when I open the `Qutebrowser.app`: "**Editor command failed to start: 'alacritty' doesn't exist or isn't executable**". However, it work just fine if I open `qutebrowser` from within the terminal.

```
c.editor.command = [
    "alacritty",
    "-e",
    "nvim",
    "-f",
    "{file}"
]
```
So I have to resort to the following (in order for it to work):

```
c.editor.command = [
    "/usr/local/bin/alacritty",
    "-e",
    "/usr/local/bin/nvim",
    "-f",
    "{file}"
]
```

I am just trying to understand why that is? And perhaps how could I fix it?

P.S.
here's my `.zprofile`:
```
export PATH="$HOME/.local/scripts:$PATH"
export PATH="$HOME/.local/bin:$PATH"
export PATH="$HOME/.local/n/bin:$PATH"
export PATH="$HOME/.local/go/bin:$PATH"
export PATH="$HOME/go/bin:$PATH"
```

**Comment by The-Compiler** [comment, marked as answer]
See #4273 - your `.zprofile` is only relevant for zsh. However, if you run qutebrowser from the GUI, no zsh is involved.
